src/main/java/com/firstwebapp/practice/spring/Album.java

```diff
@@ -1,8 +1,8 @@
 package com.firstwebapp.practice.spring;
 import javax.persistence.*;
 
-
 @Entity
+

```

Comment:
I'm not sure why you have all of these small changes in files; you should get rid of these changes rather than committing them.